Gems/Atom/Utils/Code/Include/Atom/Utils/ImGuiPassTree.h

```diff
@@ -54,6 +56,10 @@ namespace AZ
             AZStd::string m_engineRoot;
             
             AZStd::string m_attachmentReadbackInfo;
+
+            ImGuiTextFilter m_passFilter;
+
+            AZStd::unordered_set<Name> m_filteredPassNames;
```

Comment:
This should probably not be a member variable. It's only used in the context of one frame. I think a parameter in the `UpdateFilteredPassNames` and `DrawTreeView` would make the code a bit easier to read.